internal/provisioner/bundledeployment/bundledeployment.go

```diff
@@ -291,12 +321,14 @@ func (p *bundledeploymentProvisioner) reconcile(ctx context.Context, bd *rukpakv
 	case stateNeedsInstall:
 		rel, err = cl.Install(bd.Name, p.releaseNamespace, chrt, values, func(install *action.Install) error {
 			install.CreateNamespace = false
+			install.SkipCRDs = true
 			return nil
 		},
 			// To be refactored issue https://github.com/operator-framework/rukpak/issues/534
 			func(install *action.Install) error {
 				post.cascade = install.PostRenderer
 				install.PostRenderer = post
+				install.SkipCRDs = true
```

Comment:
No need to add this twice. The setting in like 324 should suffice.

Reply:
Great, will remove. Followup for that though - why the two separate install actions?

Reply:
The one with the PostRender stuff is going to be completely factored out when that helm-operator-plugins change is available (I think it's on `helm-operator-plugins` main branch now, so we might be able to go ahead and do that).

I think the thought was to keep it separate from the other install configurations to make the refactoring a clean removal of that second inline function.

But if we want to combine into one right now, that's also fine with me.